scripts/change_url

```diff
@@ -62,6 +70,7 @@ then
 	# Delete file checksum for the old conf file location
 	ynh_delete_file_checksum --file="$nginx_conf_path"
 	ynh_delete_file_checksum --file="$nginx_extra_conf_dir/php.conf"
+	#ynh_delete_file_checksum --file="$nginx_extra_conf_dir/nginx-code-error.conf"
```

Comment:
Per the CI test for change_url, I think we should reenable this. Or was it failing somehow?

Reply:
True, error from my PR, I don't remember exactly the upgrade mechanism, but it's an extra config, my memory say "*above setup than PHP nginx conf*".